timescaledb/how-to-guides/install-timescaledb-toolkit.md

```diff
@@ -1,20 +1,42 @@
-# Installing TimescaleDB Toolkit
+# Install TimescaleDB Toolkit
 
-In order to use functions from the TimescaleDB Toolkit, you'll need to install 
+In order to use functions from the TimescaleDB Toolkit, you'll need to install
 it. If you are using [Timescale Forge][] to host your database, the Toolkit is already
-installed. 
+installed.
 
-On [Timescale Cloud][] you may need to run `CREATE EXTENSION timescaledb_toolkit;` 
-in each database that you need to use the functions with. 
+On [Timescale Cloud][] you may need to run `CREATE EXTENSION timescaledb_toolkit;`
+in each database that you need to use the functions with.
 
-If you already have it installed and are updating to the latest version, run 
+If you already have it installed and are updating to the latest version, run
 `ALTER EXTENSION timescaledb_toolkit UPDATE;`.
 
 ## Self-hosted install
+
 If you are hosting your own TimescaleDB database and need to install the TimescaleDB
 Toolkit first, follow the instructions provided at the GitHub repo to [install it
 from source][install-source].
 
-[Timescale Forge]: /timescale-forge/:currentVersion:/
-[Timescale Cloud]: /timescale-cloud/:currentVersion:/
-[install-source]: https://github.com/timescale/timescaledb-toolkit#-installing-from-source
\ No newline at end of file
+## What is TimescaleDB Toolkit?
+
+TimescaleDB Toolkit is an extension for TimescaleDB that gives users access to several unique "hyperfunctions" - SQL functions designed to make manipulation and analysis of time-series data in PostgreSQL simple and efficient. Hyperfunctions work on TimescaleDB hypertables, as well as regular PostgreSQL tables. All TimescaleDB Toolkit hyperfunctions
```

Comment:
hyperfunctions might want to link to either the blog post or the API docs section or both?

Reply:
Hyperfunctions work on *data in*  TimescaleDB hypertables...